service/src/main/java/org/broadinstitute/listener/relay/Utils.java

```diff
@@ -18,6 +20,15 @@ public class Utils {
   public static final String TOKEN_NAME = "LeoToken";
   public static final String SET_COOKIE_API_PATH = "setcookie";
 
+  /**
+   * Paths which should not trigger dateAccessed runtime updates. TODO: move related logic into
+   * SetDateAccessed* classes.
+   */
+  public static final List<List<String>> KEEP_ALIVE_EXEMPT_PATHS =
+      List.of(List.of("api", "status"), List.of("welder", "status"));
+
+  private static final int PATH_PREFIX_LENGTH = 2;
```

Comment:
Realizing now that this design is also quite brittle - I think if I were doing it right the KEEP_ALIVE_EXEMPT_PATHS might be regex patterns, like `List.of(".*",".*","api","status")` which could then be directly compared to the splitted uri path.